examples/01c-Getting-started-Pytorch.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>lets call the function evaluate -&gt; calculate_loss could be single batch or full batch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>good point, made the change!</p>